ros_buildfarm/templates/devel/devel_task.Dockerfile.em

```diff
@@ -88,8 +88,19 @@ RUN python3 -u /tmp/wrapper_scripts/apt.py update-install-clean -q -y ccache
 ))@
 
 @[if run_abichecker]@
-RUN python3 -u /tmp/wrapper_scripts/apt.py update-install-clean -q -y abi-compliance-checker python3 python3-catkin-pkg-modules python3-pip 
-RUN pip3 install -U auto_abi_checker
+@{common_deps='python3 python3-catkin-pkg-modules python3-pip'}
+@[if os_name == 'ubuntu' and short_os_code_name >= 'F']@
```

Comment:
```suggestion
@{common_deps='python3 python3-catkin-pkg-modules python3-pip'
ubuntu_before_focal = (
    'precise', 'quantal', 'raring', 'saucy',
    'trusty', 'utopic', 'vivid', 'wily',
    'xenial', 'yakkety', 'zesty', 'artful', 'bionic')
}@
@[os_name == 'ubuntu' and os_code_name not in ubuntu_before_focal]@
```

Reply:
Looks like a cleaner approach. I just reduced the list of distributions to the ones not EOL (Xenial, Bionic) and condensed the suggestions in one commit 88d5520972c47b3d816bd9a8fea1f5adfe62617e